tools/format-files.sh

```diff
@@ -0,0 +1,9 @@
+
+#!/usr/bin/env bash
+
+if ! [ -x "$(command -v clang-format)" ]; then
+  printf "\e[31mError: the command clang-format was not found\e[0m"
+  exit
+fi
+
+find .. -regex '.*\.\(h\|cpp\)' -not -path '*/plugins/*' -exec clang-format -style=file -i {} \;
```

Comment:
The exclusion pattern needs extending, i.e. for DearIMGUI and utf8cpp sources (or should those go under plugins, too?)